pages/extensions/amp_dependencies/__init__.py

```diff
@@ -0,0 +1,100 @@
+# -*- coding: utf-8 -*-
+import uuid
+from sets import Set
+
+from grow import extensions
+from grow.documents import document, static_document
+from grow.extensions import hooks
+
+class AmpDependencies(object):
+    def __init__(self, doc):
+        self._doc = doc
+        self._pod = doc.pod
+        # Used to determine where to print all dependencies
+        self._placeholder = '{}'.format(uuid.uuid4())
+        # Stores registered paths
+        self._dependencies = []
+
+    def __repr__(self):
+        return '<AmpDependencies({})>'.format(self._placeholder)
+
+    def add(self, name, version='0.1'):
```

Comment:
Would it be better to use the latest version as default instead of 0.1?
The old logic used the latest version from https://playground.amp.dev/api/amp-component-versions'
Or should we always require an explicit version.
The changes in the files all use an explicit version.

Reply:
If no explicit version is set only then the extension falls back to 0.1 now.